io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -153,15 +153,15 @@ public function channelWriteLines(WritableChannel writableChannel, string[] cont
     }
 }
 
-# Write stream of lines to a channel.
+# Write stream of lines to a channel
 # ```ballerina
 # string content = ["Hello Universe..!!", "How are you?"];
 # stream<string, io:Error> lineStream = content.toStream();
 # io:Error result = io:channelWriteLinesFromStream(writableChannel, lineStream);
 # ```
-# + writableChannel - A writable channel
-# + lineStream - A stream of lines
-# + return - Either `io:Error` or `()`
+# + writableChannel - A writable channel. The possible inputs are `io:WritableByteChannel` or `io:WritableCharacterChannel`.
+# + lineStream - A stream of lines to write
+# + return - Either an `io:Error` or the null `()` value when the writing was successful
```

Comment:
Same issue as above for the description

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8